node-server/src/database/__mocks__/dao.js

```diff
@@ -0,0 +1,42 @@
+const logger    = require('../../../../common/utils/logger')
+const testData = require('../../utils/test-data')
+
+class Dao {
+  chats
+  messages
+
+  // eslint-disable-next-line no-unused-vars
+  constructor(querier) {
+    logger.info('Mock database created')
```

Comment:
This is not printed if NODE_ENV is set to test. Maybe implement a new function such as logger.test() that can be used in testing environment?